packages/core/lib/commands/test/run.js

```diff
@@ -91,7 +72,7 @@ module.exports = async function (options) {
     inputFile: file
   });
 
-  const configuredNetwork = config.networks[config.network];
+  let configuredNetwork = config.networks[config.network];
```

Comment:
Is there some reason this was changed from `const` to `let`?  I'm not seeing one.

Reply:
Oh! I forgot to change that. Previously I was trying to alter the `configuredNetwork` later in the code for some testing purpose and so changed it from `const` to `let`. I will change it to `const` again. Thanks for pointing that out @haltman-at .